README.md

```diff
@@ -27,6 +27,119 @@ For further details about the Digital Citizenship initiative, checkout the
 [main Digital Citizenship](https://github.com/teamdigitale/cittadinanza-digitale)
 repository.
 
+## Services provided by the Digital Citizenship API
+
+### Messages
+
+Public administration agencies receives millions of requests every year from
+citizens anxious to find out about the progress of their application or whether
+a payment has been received. Citizens have to spend time on hold, which wastes
+their time and costs government a lot of money in call centre running costs.
```

Comment:
I'd replace 'and costs government a lot of money in call centre running costs' with "and costs government a lot of money in running call centres"